lms/models/lti_params.py

```diff
@@ -44,6 +46,26 @@ def from_request(cls, request):
 
         return lti_params
 
+    def serialize(self, **kwargs) -> dict:
```

Comment:
From a suggestion here.

This feels like the right place :+1:

Reply:
Yeah LTIParams is a weird one, I could argue for it to be one of `domain model stuff` as it abstracts LTI stuff for the rest of the app.

I reckon models is not the right place for it, but it's the best one right now.